src/test/javascript/spec/component/programming-assessment/programming-assessment-inline-feedback.component.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { TranslateModule } from '@ngx-translate/core';
+import { DebugElement } from '@angular/core';
+import * as chai from 'chai';
+import * as sinonChai from 'sinon-chai';
+import { ArtemisTestModule } from '../../test.module';
+import { spy } from 'sinon';
+
+import { CodeEditorTutorAssessmentInlineFeedbackComponent } from 'app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component';
+import { ArtemisProgrammingManualAssessmentModule } from 'app/exercises/programming/assess/programming-manual-assessment.module';
+import { MANUAL_ASSESSMENT_IDENTIFIER } from 'app/entities/feedback.model';
+
+chai.use(sinonChai);
+const expect = chai.expect;
+
+describe('CodeEditorTutorAssessmentInlineFeedbackComponent', () => {
+    let comp: CodeEditorTutorAssessmentInlineFeedbackComponent;
+    let fixture: ComponentFixture<CodeEditorTutorAssessmentInlineFeedbackComponent>;
+    let debugElement: DebugElement;
+    const fileName = 'testFile';
+    const codeLine = 1;
+
+    beforeEach(async () => {
+        return TestBed.configureTestingModule({
+            imports: [TranslateModule.forRoot(), ArtemisProgrammingManualAssessmentModule],
+        })
+            .overrideModule(ArtemisTestModule, { set: { declarations: [], exports: [] } })
+            .compileComponents()
+            .then(() => {
+                // Ignore console errors
+                console.error = () => {
+                    return false;
+                };
+                fixture = TestBed.createComponent(CodeEditorTutorAssessmentInlineFeedbackComponent);
+                comp = fixture.componentInstance;
+                debugElement = fixture.debugElement;
```

Comment:
`debugElement` should be removed as stated from GitHub Actions

![image](https://user-images.githubusercontent.com/33484318/94557940-8a472880-025f-11eb-978e-05e4d11cbfb8.png)

Reply:
done :)